src/repl-frontend/scala/tools/nsc/interpreter/shell/ILoop.scala

```diff
@@ -103,6 +103,9 @@ class ILoop(config: ShellConfig, inOverride: BufferedReader = null,
 
   def welcome = enversion(welcomeString)
 
+  // ignore silent sbt errors on init
+  protected def isSbt: Boolean = false
```

Comment:
Indeed, in 2.12.x it was overridden in InterpreterLoop but InterpreterLoop no longer extends ILoop.